### PR TITLE
Correct the comment describing the allocated ranges of node flag bits

### PR DESCRIPTION
**Pull request by jdmpapin**
Also remove an unused flag that would have complicated the description.

### CODE REVIEW

File: compiler/il/OMRNode.hpp [hunk 1]
```diff
@@ -1877,16 +1873,17 @@ class OMR_EXTENSIBLE Node
 protected:
    enum
       {
-      // Here's map of how the flag bits have been allocated
+      // Here's map of how the flag bits have been allocated (inclusive ranges):
```
Comment (0xdaryl) [inline compiler/il/OMRNode.hpp:1876]: Since you're changing this line, how about the less folksy:
```
// Flag bits allocation map (inclusive ranges)
```

**Comment by jdmpapin** [comment]
Changed the comment's introductory line to the one suggested in review

**Comment by 0xdaryl** [comment]
Jenkins build all

**Comment by 0xdaryl** [comment]
macOS failure is #6556.  zOS is infrastructural.

**Review by 0xdaryl — APPROVED** [review]

**Pull request merged**